src/app/shared/common/json-dialog/json-dialog.component.html

```diff
@@ -42,8 +42,8 @@
     <mat-checkbox
       class="example-margin"
       [(ngModel)]="removeNulls"
-      (ngModelChange)="updateVisibleJson()"
-      >Cleaned JSON
+      (ngModelChange)="updateVisibleJson()">
+      {{ 'component_library.json' | translate }}
```

Comment:
Why not `cleaned.json`? Seeing `json` I would expect the value to be "JSON" intuitively.